curation_concerns-models/app/models/concerns/curation_concerns/file_set/derivatives.rb

```diff
@@ -16,11 +17,15 @@ def create_derivatives(filename)
         when *self.class.pdf_mime_types
           Hydra::Derivatives::PdfDerivatives.create(filename,
                                                     outputs: [{ label: :thumbnail, format: 'jpg', size: '338x493', url: derivative_url('thumbnail') }])
+          Hydra::Derivatives::FullTextExtract.create(filename,
+                                                     outputs: [{ url: uri, container: "extracted_text" }])
         when *self.class.office_document_mime_types
           Hydra::Derivatives::DocumentDerivatives.create(filename,
                                                          outputs: [{ label: :thumbnail, format: 'jpg',
                                                                      size: '200x150>',
                                                                      url: derivative_url('thumbnail') }])
+          Hydra::Derivatives::FullTextExtract.create(filename,
+                                                     outputs: [{ url: uri, container: "extracted_text" }])
         when *self.class.audio_mime_types
           Hydra::Derivatives::AudioDerivatives.create(filename,
                                                       outputs: [{ label: 'mp3', format: 'mp3', url: derivative_url('mp3') },
```

Comment:
I'm not sure what you mean by "missing class issues," but if the concerns are related to performance and stability, OK. I don't know if anyone has been relying upon the ability to extract text from A/V or image files, so in the absence of known use cases, I'm fine with punting on this. If someone has this need in the future, they can write up an issue and maybe we can try to clever our way through this.